broker/src/main/java/io/camunda/zeebe/broker/system/partitions/ZeebePartition.java

```diff
@@ -340,6 +338,7 @@ private void onInstallFailure(final Throwable error) {
           context.getPartitionId(),
           error.getMessage());
     } else {
+      LOG.warn("Failed to install partition {}", context.getPartitionId(), error);
```

Comment:
@deepthidevaki you mean this one? I guess a warning is good since we expect to recover from here right? At least this is what I understood from the code.

Reply:
We expect it to recover. But at the same time, we don't expect install to fail. If it fails there is usually a bug. I'm afraid if it is a warning we will not notice that bug an won't fix it soon.

Reply:
Ok I think I agree. 

So just to reflect:

 * We know several unrecoverable errors, which we marked as `UnrecoverableException`. In this case, we will log an error. 
 * We know several recoverable errors, which we mark as `RecoverablePartitionTransitionException`. In this case, we will just log an info level. 
 * There are certain unknown errors, which we expect to recover but we don't know for sure, since we don't know them. Currently, we log a warning and try to recover, either by stepping down or doing nothing on followers.

So for the unknown errors, the suggestion was to log at the error level. Am I right?  The issue we had with the log appender would be a known error and cause a  `RecoverablePartitionTransitionException`? Then I would agree.

Reply:
@Zelldon Yes. Your summary is spot on.

Reply:
Ok then lets do that